Documentation/op-guide/configuration.md

```diff
@@ -312,6 +312,16 @@ The security flags help to [build a secure etcd cluster][security].
 
 ## Logging flags
 
+### --logger
++ Specify 'zap' for structured logging or 'capnslog'.
++ default: capnslog
++ env variable: ETCD_LOGGER
+
+### --log-output
++ Specify 'stdout' or 'stderr' to skip journald logging even when running under systemd, or list of comma separated output targets.
```

Comment:
Similar to the `--logger` question above. Are there links to supporting documentation/examples or a full list of "output targets" that could appear and how each one behaves?

Reply:
@joelegasse Currently, we don't have example docs on etcd flags. Instead, we've only tracked what has changed with some examples here

https://github.com/coreos/etcd/blob/master/CHANGELOG-3.4.md#added-etcd

This `configuration.md` can be improved with more concrete examples.